### ISSUE TITLE
Typo3 10.4.16: An error was encountered. The plain text content could not be fetched.

### ISSUE THREAD

**Issue by ambo11** [kartolo/direct_mail#262]
since update 10.4.16 there is an error in step 2 detailed information: 	
An error was encountered.
The plain text content could not be fetched.

This is new. Back to 10.4.15 everything is working fine.

**Comment by prathers** [comment]
I can't reproduce.
Can you add some information about the error? Maybe from your sys_log?

**Comment by ambo11** [comment]
If you configure the step 2 detailed information: format html and Text - then you get the error. only html it works
in sys_log there is no error entry

**Comment by develth** [comment]
Are you on Dev-Mode with custom Domain-Name? Try to set it into Host with the IP. 
Issue should appear, if Domain could not be resolved.

**Comment by ambo11** [comment]
Hi Deveith
first of all, thanks for your answer, but I think I don't understand your answer - what is Dev-Mode - can't find something about in direct_mail docu.
I don't have an own server. I use all-inkl.com for website hosting.
